src/background/extension.ts

```diff
@@ -33,6 +31,9 @@ export class Extension {
     tabs: TabManager;
     user: UserStorage;
 
+    private isEnabledNow: boolean = null;
```

Comment:
Maybe you could remove `isEnabledNow`? Is it an optimisation? I think it makes code harder to understand and to keep track of its state. Or at least remove to something like `isEnabledByAutomation`.

Reply:
Yes, `isEnabledNow` is just an optimisation for avoiding to evaluate these interval checks repeatedly. This is still useful for `getTabMessage()`:

https://github.com/darkreader/darkreader/blob/0301db28e7e6c461033d56dd18c8b47d609b8704/src/background/extension.ts#L414-L417

As far as I know, when user toggles extension ON, `getTabMessage` calls `isEnabled()` for every tab, which might be a lot.

Reply:
I'll bundle this rename together with `isInTimeInterval` PR.